.vs/CMakeWorkspaceSettings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I have a feeling that this is something that VSCode made and isn't related to this PR. Please remove.